src/api/spec/components/previews/accordion_reviews_component_preview.rb

```diff
@@ -4,6 +4,8 @@ def preview
     pending_reviews = Review.opened.take(3)
     accepted_reviews = Review.accepted.take(2)
     declined_review = Review.declined.take(1)
-    render(AccordionReviewsComponent.new(pending_reviews + accepted_reviews + declined_review, :review, can_handle_request: true))
+    request_reviews = accepted_reviews + pending_reviews + declined_review
+    bs_request = BsRequest.last || FactoryBot.create(:bs_request_with_submit_action)
+    render(AccordionReviewsComponent.new(request_reviews, bs_request))
```

Comment:
The accordion is not visible. I'm using a re-created database filled with the `test_data` task.

The reason is in [the policy](https://github.com/openSUSE/open-build-service/blob/03455d0304537e9cc657360e5c88391730a0846b/src/api/app/policies/bs_request_policy.rb#L12) as the Admin user is not the maintainer related to whatever request this is taking.

I think we should handle this in another way. Using factories to create the values is a good practice.

Reply:
Hm, right, this is a problem with the previews using data from the development environment... My environment is now different from the one from `dev:test_data:create`...

Let me fix it